src/commands/api-mesh/status.js

```diff
@@ -35,7 +42,58 @@ class StatusCommand extends Command {
 
 		if (meshId) {
 			try {
+				const { showCloudflareURL: showEdgeMeshUrl } = await getTenantFeatures(imsOrgCode);
 				const mesh = await getMesh(imsOrgId, projectId, workspaceId, workspaceName, meshId);
+				if (showEdgeMeshUrl) {
+					const meshDeployments = await getMeshDeployments(
+						imsOrgCode,
+						projectId,
+						workspaceId,
+						meshId,
+					);
+					this.log(chalk.blackBright('Edge Mesh Status:'));
+					switch (String(meshDeployments.status).toLowerCase()) {
+						case 'success':
+							this.log(
+								'******************************************************************************************************',
+							);
+							this.log('Your mesh has been successfully built.');
+							this.log(
+								'******************************************************************************************************',
+							);
+							break;
+						case 'provisioning':
+							this.log(
+								'******************************************************************************************************',
+							);
+							this.log('Your mesh is provisioning.');
+							this.log(
+								'******************************************************************************************************',
+							);
+							break;
+						case 'error':
+							this.log(
+								'******************************************************************************************************',
+							);
+							this.log('Your mesh errored out with the following error. ', meshDeployments.error);
+							this.log(
+								'******************************************************************************************************',
+							);
+							break;
+						default:
+							//Case when deployments table has no status and only initial metadata
+							this.log(
+								'******************************************************************************************************',
+							);
+							this.log(
+								'Your mesh is not having any status. Please wait for a while and try again.',
```

Comment:
This messaging is grammatically incorrect. I would message Naomi to get proper messaging on this

Reply:
Have updated the messages as per the suggestion of Jared.